src/App.css

```diff
@@ -48,14 +48,166 @@ input {
   box-shadow: 1px 1px 2px black;
 }
 
-button {
+.homeWrap {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-color: lightblue;
+  height: 50vh;
+  width: 30vw;
+  justify-content: space-around;
+
+}
+
+.joinLink {
+  text-decoration: none;
+  color: black;
+  margin: 3vh;
+  padding: 1vh;
+  border: 2px solid black;
+  border-radius: 2px;
+  background-color: white;
+  font-size: 25px;
+  width: 90%;
+}
+
+@media screen and (max-width: 700px) {
+  .joinLink {
+    font-size: 10px;
+  }
+
+  .joinLink {
+    font-size: 11px;
+  }
+}
+
+.joinLink:hover {
+  font-size: 26px;
+  box-shadow: 1px 1px 2px black;
+  
+
+}
+
+.newListLink {
+  color: black;
+  text-decoration: none;
+  border: 2px solid black;
+  margin: 1vh;
+  border-radius: 2px;
+  background-color: white;
+  padding: 1vh;
+  position: relative;
+  top: 10vh;
+  left: -20vw;
+
+}
+
+.newListLink:hover {
+  box-shadow: 1px 1px 2px black;
+}
+
+.joinListButton {
+  color: black;
+}
+
+.tokenButton {
+  position: relative;
+  background-color: white;
+  border: 2px solid black;
+  padding: 1vh;
+  margin: 1vh;
+  height: 10vh;
+  border-radius: 2px;
+}
+
+.tokenButton:hover {
+  font-size: 32px;
+  box-shadow: 1px 1px 2px black;
+  transition: .1s;
+  transition-timing-function: ease-in;
+}
+
+.addForm {
+  border: 1px solid black;
+  justify-content: center;
+  align-items: center;
+  width: 60vw;
+  padding: 2vh;
+  height: 50vh;
+  background-color: lightblue;
+}
+
+.addItemButton {
   background-color: white;
   border: 2px solid black;
   padding: 1vh;
   margin: 1vh;
+  height: 10vh;
 }
 
-button:hover {
-  font-size: 23px;
+.addItemButton:hover {
+  font-size: 32px;
   box-shadow: 1px 1px 2px black;
+  transition: .1s;
+  transition-timing-function: ease-in;
+}
+
+.itemInputField {
+  color: black;
+  width: 90%;
+}
+
+.addFormLabel {
+  color: black;
+}
+
+.Navbar {
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  justify-content: space-around;
+  width: 100vw;
+  background-color: rgb(98, 142, 156);
+}
+
+.NavItem {
+  text-decoration: none;
+  color: black;
+  font-size: 30px;
+  border: 1px solid burlywood;
+  width: 50vw;
+  display: flex;
+  justify-content: center;
+  height: 8vh;
+  
+}
+
+.NavItem:hover {
+  font-size: 32px;
+  box-shadow: inset 0px 0px 5px #c1c1c1;
+  transition: .1s;
+  transition-timing-function: ease-in;
+
+}
+
+.BackButton {
+  display: flex;
+  position: absolute;
+  top: 2vh;
+  left: 4vh;
+  
+}
+
+.BackButton_List {
+  font-size: 60px;
+  color: black;
+  text-decoration: none;
+}
+
+.BackButton_List:hover {
+  transform: scale(1.2);
+  transition: .1s;
+  transition-timing-function: ease-in;
```

Comment:
Animations are a great way to enhance your application, thanks for adding this! 👍

Reply:
I love them!